### DISCUSSION TITLE
"Select" effect, but by manualy declaring each sub button ?

### DISCUSSION THREAD

**Discussion by pbn42** [Clooos/Bubble-Card#1018] [Questions about config, custom styles and templates]
Hi,

I realy like the new "select" effect, it makes elements more compact and easy to use.
But i'd like to use them to add a "false" select choice of vertical swing for my climates.

Actually, I made a card with different buttons, each associated with it's own action : 

![image](https://github.com/user-attachments/assets/afc1ea57-76a7-4450-9617-7e4fa579a994)

Is it possible to "convert" it to a bubble select ?

Thanks a lot !

**Comment by MrBearPresident** [comment]
This should be possible with: 
- Home Assistant: [Helper inupt_select ](https://www.home-assistant.io/integrations/input_select/)
- Home Assistant : Automation 

Once you linked those succesfully, set the input_select helper as the entity of the select button.

**Reply by pbn42** [reply]
Thanks a lot for your idea !

Unfortunatly, it won't work for my needs :
- in my case, i have a different image for each "state" of my select
- in the real world, i use streamline templates to factor the code and manage each of my different climates, and i would rather avoid creating a different select for each one of them :)

But i'll keep the idea in mind, maybe for another usage :)

Thanks again
